product_status/models/product.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2020 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl)
+
+from odoo import api, fields, models
+
+
+class ProductProduct(models.Model):
+
+    _inherit = "product.product"
+
+    status = fields.Many2one(
```

Comment:
hmm shouldn't we reuse the `state_id` field instead?

Reply:
Yes, It's will be change to `state_id` in latest update